pkg/cmd/server/api/types.go

```diff
@@ -759,29 +751,38 @@ type LDAPSchemaSpecificConfig struct {
 }
 
 type RFC2307Config struct {
-	// GroupQuery holds the template for an LDAP query that returns group entries
-	GroupQuery LDAPQuery
+	// AllGroupsQuery holds the template for an LDAP query that returns group entries.
+	AllGroupsQuery LDAPQuery
```

Comment:
The query contains the attribute that, when filtered on, will return one group. I don't like this name.